src/Laravel/LINEBotServiceProvider.php

```diff
@@ -42,4 +42,18 @@ public function register()
             return new LINEBot($httpClient, ['channelSecret' => config('line-bot.channel_secret')]);
         });
     }
+
+    /**
+     * Bootstrap any application services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        if ($this->app->runningInConsole()) {
+            $this->publishes([
+                __DIR__ . '/config/line-bot.php' => config_path('line-bot.php'),
```

Comment:
Please tell me because I am not familiar with laravel.
I think `config` would be affected to `vendor:publish --tags=config`. 
This means `line-bot.php` configs would be published  with other tagged 'config'.
Is it common?

Reply:
I think you are right if we have concern about the config file.
Actually, Don't worry about it.
(I refer to another package like spatie, league, beyondcode, etc. which is popular in Laravel community)
If we don't have specify the `--provider option`
```bash
php artisan vendor:publish --tag="config"
```
By default, all package config files will be published.
If your laravel project have `config/line-bot.php` file, it will not be overwritten.

So, I think it is not a problem.
But, If we have tagged name = 'line-bot', 
We must add more documentation to tell the user to use `vendor:publish --tag=line-bot`
I think it's not common. But it's not a problem.
(Tagged name = 'line-bot', Maybe use for other purposes in the future. Ex: publish command, publish some other files., etc.)

Thank you for your concern and I Hope this answer helps you. 🙏

Reply:
Sound good it. Thanks for your explaining!